File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/CommonArgImpl.java
package de.dimedis.mobileentry.backend;

public interface CommonArgImpl extends BaseArgImpl {
    String getUserSession();

    void setUserSession(String userSession);

    String getUserSuid();

    void setUserSuid(String userSuid);

    String getUserName();

    void setUserName(String userName);

    String getFair();

    void setFair(String fair);

    String getBorder();

    void setBorder(String border);
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/CommonArgImpl.kt
package de.dimedis.mobileentry.backend

interface CommonArgImpl : BaseArgImpl {
    fun getUserSession(): String?
    fun setUserSession(userSession: String?)
    fun getUserSuid(): String?
    fun setUserSuid(userSuid: String?)
    fun getUserName(): String?
    fun setUserName(userName: String?)
    fun getFair(): String?
    fun setFair(fair: String?)
    fun getBorder(): String?
    fun setBorder(border: String?)
}